dynawo/sources/Models/CPP/ModelNetwork/DYNModelLine.cpp

```diff
@@ -1179,11 +1182,11 @@ ModelLine::evalCalculatedVars() {
 
   calculatedVars_[iSide1Num_] = std::abs(calculatedVars_[iS1ToS2Side1Num_]);
   calculatedVars_[iSide2Num_] = std::abs(calculatedVars_[iS1ToS2Side2Num_]);
-  if (modelBus1_)
```

Comment:
Why removing this test? It might change the logic?

Reply:
Actually it is not removed but replaced by the following test which test the connection and not the presence of a bus model.

```if (getConnectionState() == CLOSED || getConnectionState() == CLOSED_1)```

Indeed, if you look at the newly added nrt IEEE14_StartFromDump, when when we start from a dump file after having disconnected a line in the initial simulation if we kepp the `if(modelBus1_)` test the side 1 inital voltage evaluated at the bus 1 voltage (different from 0) whereas it is supposed to be 0 since the line is disconnected (see below)

![image](https://github.com/user-attachments/assets/cba53b94-0245-46f5-997e-c38c0d605da4)

Reply:
ok, so to be sure I understand this well: you are using the initial iidm in the simulation that starts from the dump, right? otherwise the modelBus1 should be null